R/dbFetch.R

```diff
@@ -51,5 +51,5 @@
 #' @export
 setGeneric("dbFetch",
   def = function(res, n = -1, ...) standardGeneric("dbFetch"),
-  valueClass = "data.frame"
+  valueClass = c("data.frame", "ArrowTabular")
```

Comment:
Let's avoid touching existing generics, please.

Reply:
This is now resolved. Calling `dbFetch()` on an arrow result will return a `data.frame`.

Reply:
Yes.